src/Symfony/Bundle/DebugBundle/Resources/config/services.xml

```diff
@@ -30,6 +30,19 @@
             <argument>null</argument><!-- debug.dump_destination -->
             <argument>%kernel.charset%</argument>
         </service>
+
+        <service id="exception_flattener" class="Symfony\Component\Debug\ExceptionFlattener">
+            <argument type="collection" />
+        </service>
+
+        <service id="debug.http_processor" class="Symfony\Component\HttpKernel\Debug\HttpFlattenExceptionProcessor" public="false">
+            <tag name="exception.processor" />
+        </service>
+
+        <service id="debug.arguments_processor" class="Symfony\Component\Debug\ArgumentsFlattenExceptionProcessor" public="false">
```

Comment:
I would name them ``debug.exception_processor.*`` instead, to group related services